.github/workflows/mlir-tensorrt-ci.yml

```diff
@@ -0,0 +1,227 @@
+name: MLIR-TensorRT CI
+
+on:
+  pull_request:
+    branches:
+      - main
+    #types: [synchronize, opened, reopened, ready_for_review]
+    #paths: ['mlir-tensorrt/**']
+
+env:
+  DEFAULT_IMAGE: ghcr.io/nvidia/tensorrt-incubator/mlir-tensorrt:cuda12.5-ubuntu-llvm17
+  ADHOC_TEST_IMAGE: adhoc-test-image:cuda12.5-ubuntu-llvm17
+  REGISTRY: ghcr.io
+
+jobs:
+  mlir-tensorrt-tests:
+    if: github.event.pull_request.draft == false
+    runs-on: tripy-self-hosted
```

Comment:
I thought the idea was to use GitHub default runner for this pipeline?

Reply:
It was just for test. We can't use GPU's there since driver version is less for CUDA>=12.5 we need.